node/parachain/src/chain_spec.rs

```diff
@@ -40,12 +40,14 @@ pub struct Extensions {
 
 impl Extensions {
 	/// Try to get the extension from the given `ChainSpec`.
+	#[allow(clippy::borrowed_box)]
 	pub fn try_get(chain_spec: &Box<dyn sc_service::ChainSpec>) -> Option<&Self> {
 		sc_chain_spec::get_extension(chain_spec.extensions())
 	}
 }
 
 pub fn get_chain_spec(para_id: ParaId) -> Result<ChainSpec, String> {
+	#[allow(clippy::or_fun_call)]
 	let wasm_binary = WASM_BINARY.ok_or("Development wasm binary not available".to_string())?;
```

Comment:
```rust
let wasm_binary = WASM_BINARY.ok_or_else(|| "Wasm binary not available".to_string())?;
```

Reply:
I suggested the same change upstream https://github.com/paritytech/substrate/pull/7802